device/celestica/x86_64-cel_seastone-r0/port-mode.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
The script was moved to sonic-utilities repo, a PR was generated there:
https://github.com/Azure/sonic-utilities/pull/87